src/orbs/Client.ts

```diff
@@ -45,6 +48,19 @@ export class Client {
     return [req, Encoding.encodeHex(rawTxId)];
   }
 
+  createDeployTransaction(publicKey: Uint8Array, privateKey: Uint8Array, contractName: string, processorType: number, ...sources: Uint8Array[]): [Uint8Array, string] {
+    const inputArguments: Argument[] = [
```

Comment:
If you choose to , you can do it all in one line. (Not sure it's readable, up to you)

```ts
const inputArguments: Argument[] = [
  argString(contractName),
  argUint32(processorType),
  ...sources.map(argBytes),
];
```

Reply:
Good catch!